doc/source/user_guide/adaptive_templates.rst

```diff
@@ -342,82 +328,70 @@ The ``triangular_notch()`` function requires 4 arguments:
-must be calculated in the Adaptive Templates Script and
-specified.
+where the arguments ``rotor_radius``, ``notch_angular_width``, ``notch_centre_angle`` and
+``notch_depth`` must be calculated in the Adaptive Templates Script and specified.
 
-The notch region properties can then be defined and
-the region can be set in Motor-CAD,
-as described earlier in this guide.
+The notch region properties can then be defined and the region can be set in Motor-CAD, as described
+earlier in this guide.
 
-For a full Adaptive Templates example
-using the workflow described here,
-see :ref:`ref_BPM_Triangular_Rotor_Notches`.
+For a full Adaptive Templates example using the workflow described here, see
+:ref:`ref_BPM_Triangular_Rotor_Notches`.
 
-Details on the Geometry Shapes functions within ``ansys.motorcad.core.geometry_shapes``
-are available in the :ref:`ref_API_Reference` under :ref:`ref_geometry_shapes`.
+Details on the Geometry Shapes functions within ``ansys.motorcad.core.geometry_shapes`` are
+available in the :ref:`ref_API_Reference` under :ref:`ref_geometry_shapes`.
 
 Creating and modifying adaptive templates scripts
 *************************************************
 
-It is recommended to create Adaptive Template Scripts outside Motor-CAD,
-using a Python Integrated Development Environment (IDE) (such as PyCharm).
-Using an IDE allows for faster creation of the script,
-allowing access to autocompletion, code correction
-and other features which are not available in the Motor-CAD scripting interface.
+It is recommended to create Adaptive Template Scripts outside Motor-CAD, using a Python Integrated
+Development Environment (IDE) (such as PyCharm). Using an IDE allows for faster creation of the
+script, allowing access to autocompletion, code correction and other features which are not
+available in the Motor-CAD scripting interface.
 
-This is essential when writing complex scripts,
-allowing issues with the script to be fixed
-and the inspection of Python objects
-(for example geometry regions from Motor-CAD).
+This is essential when writing complex scripts, allowing issues with the script to be fixed and the
+inspection of Python objects (for example geometry regions from Motor-CAD).
 
-For more information on the Synchronous Reluctance machine geometry
-with curved flux barriers used for this example,
-please see :ref:`ref_SYNC_Curve_Flux_Barriers`.
+For more information on the Synchronous Reluctance machine geometry with curved flux barriers used
+for this example, please see :ref:`ref_SYNC_Curve_Flux_Barriers`.
 
 Working on the adaptive templates script
 ----------------------------------------
-Adaptive templates script can be edited from an external IDE (for example PyCharm, VSCode).
-When using an external IDE it is important to ensure that the script contains this method
-before getting/setting any Motor-CAD geometry:
+Adaptive templates script can be edited from an external IDE (for example PyCharm, VSCode). When
```

Comment:
Now here "templates" is not capitalized, where earlier it was. We should be using the same term consistently. If Adaptive Templates is not a proper name, use lower case throughout.